apis/nucleus/spec/spec.conf.js

```diff
@@ -12,19 +12,19 @@ module.exports = {
   },
   parse: {
     types: {
-      'enigma.Doc': {
+      'EngineAPI.IApp': {
         url: 'https://github.com/qlik-oss/enigma.js/blob/master/docs/api.md#generated-api',
       },
-      'qae.GenericObjectLayout': {
+      'EngineAPI.IGenericObjectLayout': {
         url: 'https://core.qlik.com/services/qix-engine/apis/qix/definitions/#genericobjectlayout',
```

Comment:
While we are touching this, we can update these links, which are not working, as well. So instead of pointing towards core.qlik... we can point this to https://qlik.dev/apis/json-rpc/qix/schemas#%23%2Fdefinitions%2Fschemas%2Fentries%2FGenericObjectLayout for instance (since qlik core has been deprecated).

Reply:
I looked for these urls but couldn't find any suitable ones. I'll up them!